### PR TITLE
Unity minor version and package update

### PR DESCRIPTION
**Pull request by andybak**
This is probably a good time to bump the latest 2019.4 and update a few packages.

### CODE REVIEW

**Review by mikeage — CHANGES_REQUESTED** [review]
Please update the GitHub Actions yaml as well

**Comment by mikeskydev** [comment]
Should anything related to quicksearch be committed? there's also a UserSettings folder that might need to be gitignored

**Comment by andybak** [comment]
> Should anything related to quicksearch be committed? there's also a UserSettings folder that might need to be gitignored

They just seem to be config files. You're right about UserSettings though. I've removed that file gitignored the directory.

**Comment by andybak** [comment]
Can anyone see what's wrong with the build? It's the licence file but the commit looks fine:

https://github.com/IxxyXR/open-brush/commit/4061e88a9041dd37a6cf9f907704100d2918a35d

I'll retry the build just in case it was a remote error.

**Comment by mikeage** [comment]
OK, I know what's wrong. I'm on my mobile now, will write more later. We need to update the license (fallback) to match the unity version.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -11,12 +11,12 @@ on:  # yamllint disable-line rule:truthy
...
...\"UnityPersonal\" Type=\"EDITOR\" ValidTo=\"9999-12-31T00:00:00\"/>\n            <Entitlement Ns=\"unity_editor\" Tag=\"DarkSkin\" Type=\"EDITOR_FEATURE\" ValidTo=\"9999-12-31T00:00:00\"/>\n        </Entitlements>\n    </License>\n<Signature xmlns=\"http://www.w3.org/2000/09/xmldsig#\"><SignedInfo><CanonicalizationMethod Algorithm=\"http://www.w3.org/TR/2001/REC-xml-c14n-20010315#WithComments\"/><SignatureMethod Algorithm=\"http://www.w3.org/2000/09/xmldsig#rsa-sha1\"/><Reference URI=\"#Terms\"><Transforms><Transform Algorithm=\"http://www.w3.org/2000/09/xmldsig#enveloped-signature\"/></Transforms><DigestMethod Algorithm=\"http://www.w3.org/2000/09/xmldsig#sha1\"/><DigestValue>xCBKYap0JQVBqX22jba4k5Da5t0=</DigestValue></Reference></SignedInfo><SignatureValue>n8eYH6Exi6mpw6onE5blxgQY1bAb+NLrsblMlFLUTPVTGw5KmugyVHedyZiI47D3nZVK/FpGaVxJ\n2xcNBr8yR1lfQrakloAZhvlCTiNi/O8N+6aHGZgyi/5+2SLqj0E/ZB+bVlTaSioLy/cF6iOt0d0T\nvW+VEdhht9iWrKKeNBakcXGccX5tQhJFaTR8aYevp0t0TMz3cfGPhs5qDivufqoFl34Y6qa8HjAC\n+OvrlD0TIgMyev+czK0KMIRsT/euOoyOHupXb93BuiMWceRIqf4mYvPKJbhx15Zy3uxHbsF6lV8q\nGah5JCb8iGRTDmG2JzMg6pXCjiznPFNk4Tkl7Q==</SignatureValue></Signature></root>", null]')[secrets.UNITY_SERIAL != null] }}
```
Comment (mikeage) [inline .github/workflows/build.yml:19]: The license was generated with 2019.4.25f1, so we need to keep that part. It's still valid for 2019.4.33f1 though (in earlier PRs, I'd updated the license with each unity version update, but it turns out that this shouldn't be necessary.)

In other words, revert this change, and just leave line 14. 

(confirmation: https://discord.com/channels/710946343828455455/710946344264794219/917566961054007308, from the game-ci discord server)

**Review by mikeage — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -216,7 +216,7 @@ jobs:
 
       - name: Install TextMesh Pro package
         run: |
-          # version 2.1.4; must be updated if the version changes
+          # version 2.1.6; must be updated if the version changes
           # This replaces the GUI's "Window -> TextMesh Pro -> Import TMP Essential Resources". I don't know why Unity makes this sort of thing so hard!
           mkdir tmp.plugin
           wget -q https://download.packages.unity.com/com.unity.textmeshpro/-/com.unity.textmeshpro-2.1.4.tgz -O tmp.plugin/plugin.tgz
```
Comment (mikeage) [inline .github/workflows/build.yml:222]: can you update this line please? :-)

**Comment by andybak** [comment]
Let's forget this - the 2020 update is on the horizon.

**Pull request closed without merging**